src/v2/components/Validators/index.jsx

```diff
@@ -0,0 +1,95 @@
+// @flow
+import {Container, Grid} from '@material-ui/core';
+import React from 'react';
+import {observer} from 'mobx-react-lite';
+import {map} from 'lodash/fp';
+import Card from 'v2/components/UI/StatCard';
+import SectionHeader from 'v2/components/UI/SectionHeader';
+import NodesStore from 'v2/stores/nodes';
+
+import Button from '../UI/Button';
+import ValidatorsMap from './ValidatorsMap';
+import ValidatorsTable from './Table';
+import useStyles from './styles';
+
+const Validators = () => {
+  const classes = useStyles();
+  const {cluster, clusterChanges} = NodesStore;
+  const cards = [
+    {
+      title: 'Total Circulating SOL',
+      value: cluster.supply / Math.pow(2, 34).toFixed(2),
+      changes: clusterChanges.supply,
+      period: 'since yesterday',
+    },
+    {
+      title: 'Total Bonded Tokens',
+      value: '100,000',
```

Comment:
This data is available as the sum of all `.stake` values in the cluster nodes voteAccount objects

Reply:
ok. this one i need to work on. don't request this data yet.